upfadapter/config/config.go

```diff
@@ -30,11 +32,11 @@ const (
 // UPF structure
 type UPNode struct {
 	UpfName     string
-	NodeID      pfcpType.NodeID
+	NodeID      types.NodeID
```

Comment:
Would it make sense to use the one from the `smf/context`?

Reply:
I'll resolve this comment so we have the conversation on the other comment that I already answered to.